newsroom/wire/views.py

```diff
@@ -56,7 +56,8 @@
 
 from .search import get_bookmarks_count
 from .items import get_items_for_dashboard
-from ..upload import ASSETS_RESOURCE, get_upload
+from newsroom.assets.module import ASSETS_RESOURCE
```

Comment:
If we include `__all__ = ["ASSETS_RESOURCE", "get_upload", ...]` etc into the `newsroom.assets.__init__.py` file, then we can provide an interface into the Assets module and expose only what other modules need.

Then we can do the following:
`from newsroom.asssets import ASSETS_RESOURCE, get_upload`